conformance/tests/admin-network-policy-core-priority.go

```diff
@@ -0,0 +1,144 @@
+				clientPod.Status.PodIP, int32(80), s.TimeoutConfig.RequestTimeout, false)
+			assert.Equal(t, true, success)
+			// draco-malfoy-1 is our client pod in slytherin namespace
+			success = kubernetes.PokeServer(t, "network-policy-conformance-slytherin", "draco-malfoy-1", "tcp",
+				clientPod.Status.PodIP, int32(8080), s.TimeoutConfig.RequestTimeout, false)
+			assert.Equal(t, true, success)
+		})
+
+		t.Run("Should Deny traffic to slytherin from gryffindor respecting ANP", func(t *testing.T) {
+			ctx, cancel := context.WithTimeout(context.Background(), s.TimeoutConfig.GetTimeout)
+			defer cancel()
+			// This test uses `priority-50-example` ANP; takes precedence over old-priority-60-new-priority-40-example ANP
+			// draco-malfoy-0 is our server pod in slytherin namespace
+			clientPod := &v1.Pod{}
+			err := s.Client.Get(ctx, client.ObjectKey{
+				Namespace: "network-policy-conformance-slytherin",
+				Name:      "draco-malfoy-0",
+			}, clientPod)
+			framework.ExpectNoError(err, "unable to fetch the server pod")
+			// harry-potter-0 is our client pod in gryffindor namespace
+			// ensure ingress is DENIED to gryffindor from slytherin
+			// egressRule at index0 will take effect
+			success := kubernetes.PokeServer(t, "network-policy-conformance-gryffindor", "harry-potter-0", "tcp",
+				clientPod.Status.PodIP, int32(80), s.TimeoutConfig.RequestTimeout, false)
+			assert.Equal(t, true, success)
+			// harry-potter-1 is our client pod in gryffindor namespace
+			success = kubernetes.PokeServer(t, "network-policy-conformance-gryffindor", "harry-potter-1", "tcp",
+				clientPod.Status.PodIP, int32(8080), s.TimeoutConfig.RequestTimeout, false)
+			assert.Equal(t, true, success)
+		})
+
+		t.Run("Should respect ANP priority field; thus passing both ingress and egress traffic over to BANP", func(t *testing.T) {
+			ctx, cancel := context.WithTimeout(context.Background(), s.TimeoutConfig.GetTimeout)
+			defer cancel()
+			// This test uses `old-priority-60-new-priority-40-example` ANP
+			anp := &v1alpha1.AdminNetworkPolicy{}
+			err := s.Client.Get(ctx, client.ObjectKey{
+				Name: "old-priority-60-new-priority-40-example",
+			}, anp)
+			framework.ExpectNoError(err, "unable to fetch the admin network policy")
+			// change priority from 60 to 40
+			anp.Spec.Priority = 40
+			err = s.Client.Update(ctx, anp)
+			framework.ExpectNoError(err, "unable to update the admin network policy")
+			// harry-potter-0 is our server pod in gryffindor namespace
+			clientPod := &v1.Pod{}
+			err = s.Client.Get(ctx, client.ObjectKey{
+				Namespace: "network-policy-conformance-gryffindor",
+				Name:      "harry-potter-0",
+			}, clientPod)
+			framework.ExpectNoError(err, "unable to fetch the server pod")
+			// draco-malfoy-0 is our client pod in slytherin namespace
+			// ensure ingress is PASSED to gryffindor from slytherin - the baseline admin network policy ALLOW should take effect
```

Comment:
Can we really tell the difference here between PASS and ALLOW?

Reply:
I guess you could eliminate 3. with and large-scale baseline deny

Reply:
yeah measuring pass and allow has been tricky actually.. good point here.
if there is something to catch this underneath like we do in the integration tests: https://github.com/kubernetes-sigs/network-policy-api/blob/fdf177d46260fb736d44a68aa8ca213702a50481/conformance/tests/admin-network-policy-core-integration.go#L127 then we have a proper check, otherwise the two pretty much behave the exact same way.